facts.json

```diff
@@ -11,6 +11,15 @@
     "The Body Effect is modeled as a change in the Threshold voltage",
     "The NMOS and PMOS are two common types of MOSFETs",
     "For small signal AC analysis of a MOSFET, either the pi model or the T model can be used",
```

Comment:
capitalization in 'pi model' vs 'T model' feels weird to me but if thats that, u can leave it bc 'Pi model' feels worse